select_multiple_field/forms.py

```diff
@@ -57,7 +57,12 @@ def to_python(self, value):
         Method also handles lists and strings
         """
         if (value == self.empty_value) or (value in self.empty_values):
-            return self.empty_value
+            try:
+                # must be iterable
+                it = iter(self.empty_value)
+                return self.empty_value
+            except TypeError, te:
```

Comment:
This won't work with Python 3, use instead: `except TypeError as te:`

But since `te` isn't used at all: `except TypeError:` ?